tests/stack_test.go

```diff
@@ -561,9 +561,9 @@ func stackFileFormatAsserters(t *testing.T, e *ptesting.Environment, stackName s
 		}
 		if t.Failed() {
 			fmt.Printf("Stacks dir state at time of failure (gzip: %t):\n", gzip)
-			files, _ := ioutil.ReadDir(e.CWD + "/" + stacksDir)
+			files, _ := os.ReadDir(e.CWD + "/" + stacksDir)
 			for _, file := range files {
-				fmt.Println(file.Name(), file.Size())
```

Comment:
This could use `file.Info().Size()` right?

Reply:
@Frassle Yes, you are right. But we need to do an extra error handling before `file.Info()`  returns `(FileInfo, error)`